monai/data/image_reader.py

```diff
@@ -604,7 +602,7 @@ def get_data(self, img):
         for i in ensure_tuple(img):
             header = self._get_meta_dict(i)
             header["spatial_shape"] = self._get_spatial_shape(i)
-            data = np.asarray(i)
+            data = np.moveaxis(np.asarray(i), 0, 1)
```

Comment:
May I know why we also need to change the axis in PIL reader here?

Thanks.

Reply:
itk and pil uses one way to run the array index, nibabel and numpy uses another way...